WordPress/src/main/res/layout/theme_browser_fragment.xml

```diff
@@ -8,7 +8,7 @@
                 android:layout_height="match_parent">
 
     <org.wordpress.android.ui.ActionableEmptyView
-        android:id="@+id/actionable_empty_view"
+        android:id="@+id/no_matches_view"
```

Comment:
I know that `no_matches_view` and `no_themes_view`/`no_themes_text` below makes more sense in the context of themes fragment, but customary (and for convenience) we are mostly using generic names for empty views, like `actionable_empty_view` (since we use them a lot, and it makes it easy to reference them all around the place) :)

Reply:
Sure thing. I have reverted the name changes.

> makes it easy to reference them all around the place

Would be curious to know a bit more about this. Obviously, my inclination was to go the other way and to use names specific to this fragment. I'd like to learn about what I maybe wasn't considering when I made that change, so if there were any specific considerations that led us to go the other direction, I'd really like to learn more about them. 😀

Reply:
I wish I could give you a good answer on why we are doing it, but I don't have one :D Actionable empty views were added to the app relatively recently, and we added them with the same ID for reasons no one probably remembers :)